buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -258,9 +259,12 @@ class ClusterFormationTasks {
                 }
             }
             File destConfigFile = new File(node.homeDir, 'config/' + extraConfigFile.getKey())
-            copyConfig.into(destConfigFile.canonicalFile.parentFile)
-                      .from({ extraConfigFile.getValue() }) // wrap in closure to delay resolution to execution time
-                      .rename { destConfigFile.name }
+            // wrap source file in closure to delay resolution to execution time
+            copyConfig.from({ extraConfigFile.getValue() }) {
+                // this must be in a closure so it is only applied to the single file specified in from above
```

Comment:
The line does contain a closure. Notice the curlies inside the parens for `from`. :)